LICENSE

```diff
@@ -200,3 +200,90 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+---
+
+This product bundles Sqlline (https://github.com/julianhyde/sqlline)
+which is licensed under the 3-clause BSD license
+
+Copyright (c) 2002,2003,2004,2005,2006,2007 Marc Prud'hommeaux
+Copyright (c) 2004-2010 The Eigenbase Project
+Copyright (c) 2013-2014 Julian Hyde
+All rights reserved.
+
+---
+
+This product bundles portions of AngularJS (https://angularjs.org/) which
```

Comment:
The only reason we bundle this project and the ones below it are for the trace UI. If this is going to make it more difficult to do a binary release or require more work for the source releases, I think we should just remove the trace UI from both source and binary releases.